src/Themes/businessDashboard/foundations/typography.st.css

```diff
@@ -0,0 +1,30 @@
+@st-global-custom-property --wsr-font-family, --wsr-font-weight-bold;
+
+:import {
+  -st-from: "../../../Foundation/stylable/typography.st.css";
+  -st-named: heading-h3, heading-h4, heading-h5, weight-mixin, font-mixin;
```

Comment:
From the stylable docs, you can rename the variables when imported. this means you are not bound to the variables and your tokens can be called `heading-h*`

```suggestion
  -st-named: heading-h3 as heading-h3-original, heading-h4 as heading-h4--original, heading-h5 as heading-h5-original, weight-mixin, font-mixin;
```


![image](https://user-images.githubusercontent.com/6093192/97810869-a0da0880-1c7f-11eb-9a96-74c1a641d34d.png)

Reply:
thanks! it looks like I don't need it at all following your previous suggestions